services/console-proxy/rdpconsole/src/main/java/common/Client.java

```diff
@@ -47,6 +47,10 @@ enum Protocol {
         NONE, VNC, RDP, HYPERV
     }
 
+    private static final String VNC = "vnc";
+    private static final String RDP = "rdp";
+    private static final String HYPERV = "hyperv";
```

Comment:
i think, no need of this string definitions, try with string to enum constant conversion?